bin/sign.py

```diff
@@ -38,19 +38,22 @@ def sign_operation(key_code: str, operation: str) -> JSON:
         "ToolVersion": "1.0",
     }
 
+
 def sign_operation_linux(key_code: str, operation: str) -> JSON:
     return {
-        "KeyCode" : key_code,
-        "OperationCode" : operation,
-        "Parameters" : {},
-        "ToolName" : "sign",
-        "ToolVersion" : "1.0"
+        "KeyCode": key_code,
+        "OperationCode": operation,
+        "Parameters": {},
+        "ToolName": "sign",
+        "ToolVersion": "1.0",
     }
 
+
 def linux_sign(key_code: str) -> JSON:

```

Comment:
[nit]: extra line added

Reply:
I added this line for consistency. All the methods in this file are preceded by two lines.

Reply:
@shalinikhare27: That extra newline is the norm in Python. It's a community "standard".